renku/cli/dataset.py

```diff
@@ -25,6 +25,29 @@
 .. code-block:: console
 
     $ renku dataset create my-dataset
+    Creating a dataset ... OK
+
+Listing all datasets:
+
+.. code-block:: console
+
+    $ renku dataset
+    ID        NAME           CREATED              AUTHORS
+    --------  -------------  -------------------  ---------
+    0ad1cb9a  some-dataset   2019-03-19 16:39:46  sam
+    9436e36c  my-dataset     2019-02-28 16:48:09  sam
+
+Deleting a dataset:
+
+.. code-block:: console
+
+    $ renku dataset rm some-dataset
+    OK
```

Comment:
We should update the message to be consistent with `dataset create`

```suggestion
    Removing dataset(s) <UUID>, [<UUID>, ...] ... OK
```

Reply:
Is this an issue of the documentation or of implementation? If the documentation matches what is implemented, I suggest we leave it until the implementation changes.

Reply:
On all recently added commands, the output is OK if the operation user requested is executed successfully or nothing printed at all. So instead of having the output like proposed, I would just align the output of dataset creation to be the same.

Reply:
We can completely remove a success messages and rely only on the exit code.